tests/util_test/util.hpp

```diff
@@ -103,6 +103,15 @@ inline bool wait(wait_opts const& opts, function<void(wait_ctx&)> const& poller)
   }                                         \
   assert(true)  // to justify trailing semicolon
 
+#define WAIT_EXPECT_NE(ctx, o1, o2)         \
+  if (o1 == o2) {                           \
+    if (ctx.fail(); !ctx.is_last_attempt) { \
+      return;                               \
+    }                                       \
+    EXPECT_NE(o1, o2);                      \
+  }                                         \
+  assert(true)  // to justify trailing semicolon
```

Comment:
I don't get why need assert here. Looks like assert(true) will never fail

Reply:
I will fix it, it is just for semicolon